src/views/solicitacao-de-financiamento/index.tsx

```diff
@@ -0,0 +1,205 @@
+import React from "react";
+import Button from "src/components/button";
+import Input from "src/components/input";
+import { Layout } from "src/components/layout"
+import * as S from "./styles"
+import { VscCloudUpload } from "react-icons/vsc"
+import { IoMdArrowDropdown } from "react-icons/io"
+
+interface PageProps {
+    hideFooter?: boolean;
+    isLogged?: boolean;
+}
+
+const SolicitacaoDeFinanciamentoView:React.FC<PageProps> = (props) => {
+
+    return(
+        <Layout {...Object.assign({}, props, {hideFooter: true, isLogged: true})}>
+            <S.Container>
+                <S.Content>
+                    <S.FirstSection>
+                        <S.Title>Solicitar Financiamento</S.Title>
+                        <S.Description>
+                            Deve preencher o formulário abaixo para a solicitacão de um financiamento 
+                        </S.Description>
+                    </S.FirstSection>
+                    <S.SecondSection>
+                        <h2>Dados Pessoais</h2>
+                        <S.Separator/>
+                        <S.SectionInput>
+                            <S.ContainerInput>
+                                <S.Label>Seu primeiro nome</S.Label>
+                                <Input 
+                                    type="text"
+                                    placeholder="Nome"
+                                />
+                            </S.ContainerInput>
+                            <S.ContainerInput>
```

Comment:
A borda das Inputs não estão fiéis ao layout. Parece apenas uma linha preta, magra e feia. Não parece uma borda tranparente.